src/test/java/digkins/services/ArtifactsServiceTests.java

```diff
@@ -0,0 +1,58 @@
+package digkins.services;
+
+import com.offbytwo.jenkins.JenkinsServer;
+import com.offbytwo.jenkins.client.JenkinsHttpClient;
+import com.offbytwo.jenkins.model.*;
+import com.redhat.digkins.services.ArtifactsService;
+import org.junit.Test;
+
+import java.io.FileInputStream;
+import java.io.InputStream;
+import java.util.Arrays;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.mockito.BDDMockito.given;
```

Comment:
@wtrocki I had the old TDD style used in other tests. I think we should use one approach.
Unless you have strong opinions about BDD, I will convert `given` to `when`